CHANGELOG.md

```diff
@@ -47,6 +48,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+### Dependencies
+
+| Dependency             | Old version | New version |
+| ---------------------- | ----------- | ----------- | ----------------------------- |
+| `ascat`                | 3.0.0       | unchanged   |
+| `bcftools`             | 1.15.1      | unchanged   |
+| `bwa-mem2`             | 2.2.1       | unchanged   |
+| `bwa`                  | 0.7.17      | unchanged   |
+| `cancerit-allelecount` | 4.3.0       | unchanged   |
+| `cnvkit`               | 0.9.9       | unchanged   |
+| `control-freec`        | 11.6        | unchanged   |
+| `deepvariant`          | 1.3.0       | unchanged   |
+| `dragmap`              | 1.2.1       | unchanged   |
+| `ensembl-vep`          | 106.1       | unchanged   | //should we update this here? |
+| `fastp`                | 0.23.2      | unchanged   |
+| `fastqc`               | 0.11.9      | unchanged   |
+| `fgbio`                | 2.0.2       | unchanged   |
+| `freebayes`            | 1.3.5       | unchanged   |
+| `gatk4`                | 4.2.6.1     | 4.3.0.0     |
```

Comment:
I think we can keep only the updated ones

Reply:
more to come, see modules repo 😆